test/libsolidity/syntaxTests/parsing/mapping_with_names_struct_member_6.sol

```diff
@@ -0,0 +1,7 @@
+contract test {
+    struct Person {
+        mapping(uint nameSame => mapping(uint name1 => mapping(uint nameSame => uint name2) name3) name4) name5;
```

Comment:
Another three-level nested mapping test, but this time valid, please :)

Reply:
I've added that, please review.